index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Bus Mall</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<header>
+<nav>
+
+</nav>
+</header>
```

Comment:
You seem to have a lot of leftover HTML. When you're working off of previous examples, make sure to only bring in the code you need and that you're actually using; otherwise, this extra HTML is more likely to cause problems.